src/client/src/message.js

```diff
@@ -11,24 +11,24 @@ class Conversation {
         if(this.messages.length) {
             return this.messages[this.messages.length - 1];
         }
-        return null;
+        return '';
     }
     addMessage(message) {
-        message = Object.assign(new Message(), message);
+        message = Object.assign(new Msg(), message);
         message.sender = Object.assign(new Sender(), message.sender);
         let newSender = message.sender;
         let lastMessage = this.lastMessage;
         // return this;
-        if (lastMessage && newSender.equals(lastMessage.sender)) {
-            lastMessage.concat(message);
-        } else {
+        // if (lastMessage && newSender.equals(lastMessage.sender)) {
+        //     lastMessage.concat(message);
+        // } else {
             this.messages.push(message);
-        }
+        //}
         return this;
     }
 }
 
-class Message {
+class Msg {
```

Comment:
XD, changing my class names, so sassy. I think you just need to rename in the unit tests as well to fix the failing ones

Reply:
xD